retrofit-converters/wire/pom.xml

```diff
@@ -26,6 +26,12 @@
     <dependency>
       <groupId>com.squareup.wire</groupId>
       <artifactId>wire-runtime</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.squareup.okio</groupId>
+          <artifactId>okio</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
This really should be needed, but for some reason the older Okio dependency from Wire's runtime is beating out the newer version which is specified by OkHttp (itself a transitive dependency of Retrofit).